README.md

```diff
@@ -2,16 +2,20 @@
 
 An auth provider for [react-admin](https://github.com/marmelab/react-admin) which handles authentication via a [Auth0](https://auth0.com) instance.
 
+[![Documentation]][DocumentationLink] 
+[![Source Code]][SourceCodeLink] 
+
+[Documentation]: https://img.shields.io/badge/Documentation-green?style=for-the-badge
+[Source Code]: https://img.shields.io/badge/Source_Code-blue?style=for-the-badge
+
+[DocumentationLink]: ./packages/ra-auth-auth0/Readme.md 'Documentation'
```

Comment:
Shouldn't we put an absolute link here for NPM?

Reply:
We could, but what would be the benefit of that?